data/tilesets/buildings/unknown.license

```diff
@@ -0,0 +1,2 @@
+SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Not sure that this is a good sprite for an unknown building, the silhouette of a building would work better.

Reply:
I'm going to remove it then and we can determine what we want to do for ultimate fallback at another time.